internal/pkg/daemon/bpfrecorder/bpf/recorder.bpf.c

```diff
@@ -51,6 +53,9 @@ char LICENSE[] SEC("license") = "Dual BSD/GPL";
 #define unlikely(x) __builtin_expect((x), 0)
 #endif
 
+#define trace_hook(...)
```

Comment:
nit: this look like an empty macro, I would remove it if it is not used.

Reply:
The intent here is that one can quickly toggle between no-op and bpf_printk, which makes debugging BPF much nicer. Does that make sense, any better suggestions?